native/src/main/java/io/ballerina/stdlib/sql/parameterprocessor/AbstractResultParameterProcessor.java

```diff
@@ -214,6 +214,10 @@ public abstract Object processCustomOutParameters(
     public abstract Object processCustomTypeFromResultSet(ResultSet resultSet, int columnIndex,
                                                            ColumnDefinition columnDefinition) throws ApplicationError;
 
+    public abstract Object convertArray(String objectTypeName, Object[] dataArray, Type ballerinaType);
+
+    public abstract Object customArrayType(Object[] dataArray, Type ballerinaType) throws ApplicationError;
```

Comment:
What is this additional function?

Reply:
Removed unnecessary  method and renamed to `processCustomArray `.